build/install_linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+sudo sh bootstrap-salt.sh
+sudo apt install git-all
+git clone https://github.com/plus3it/amazon-inspector-formula
```

Comment:
Can remove this, travis handles it automatically, always. The current working dir is already the project root.